Tasks/Common/coveragepublisher/module.json

```diff
@@ -0,0 +1,8 @@
+{
+    "messages": {
+        "InstallDotNetCoreForHtmlReport": "Please install dotnet core to enable automatic generation of Html report.",
```

Comment:
Is .NET core needed only for HTML reports? Is it not needed otherwise?

Reply:
dotnet core is needed for all of the publishing, we call the log store apis through our stuff